crates/nu-command/src/commands/filters/select.rs

```diff
@@ -45,11 +49,11 @@ impl WholeStreamCommand for Command {
     }
 }
 
-fn select(args: CommandArgs) -> Result<OutputStream, ShellError> {
-    let name = args.call_info.name_tag.clone();
-    let columns: Vec<ColumnPath> = args.rest(0)?;
-    let input = args.input;
-
+fn select(
+    name: Tag,
+    columns: Vec<ColumnPath>,
+    input: InputStream,
+) -> Result<OutputStream, ShellError> {
```

Comment:
I've moved `CommandArgs` to `run()` and leave `select()` as "pure" unit-testable function, this seems to be common approach in other modules but I can revert my changes if it shouldn't be done.